### PR TITLE
fixing default encoding issues

### PR DESCRIPTION
**Pull request by ajaijoy-bmc**
Fixing default encoding issues in clouddriver.

### CODE REVIEW

File: clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesCredentials.java [hunk 1]
```diff
@@ -71,7 +72,11 @@ public KubernetesCredentials(KubernetesApiAdaptor apiAdaptor,
 
         HashMap<String, String> secretData = new HashMap<>(1);
         String dockerCfg = String.format("{ \"%s\": { \"auth\": \"%s\", \"email\": \"%s\" } }", account.getV2Endpoint(), account.getBasicAuth(), account.getEmail());
-        dockerCfg = new String(Base64.getEncoder().encode(dockerCfg.getBytes()));
+        try{
```
Comment (duftler) [inline clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesCredentials.java]: Please put single-space separators between `try` and its opening brace, before and after `catch`, and just before `catch`'s opening brace.

**Comment by duftler** [comment]
LGTM after addressing comment.

File: clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesCredentials.java (hunk 1 above)
Comment (ajaijoy) [inline clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesCredentials.java]: will do, is there a formatter that i can use going forward

Comment (duftler) [inline clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesCredentials.java]: I _think_ the default configuration resulting from using `./gradlew idea`
to generate the idea project artifacts will do what you want.

(If not using IntelliJ, I'm sure there are some other options.)

**Comment by lwander** [comment]
Thanks for doing this. Could you rebase, and squash your commits?

**Comment by ajaijoy-bmc** [comment]
will do now.

**Pull request merged**